contracts/implementations/nodes/AftermarketDevice.sol

```diff
@@ -518,6 +522,56 @@ contract AftermarketDevice is
         _setInfos(tokenId, attrInfo);
     }
 
+    /**
+     * @notice Set the device address of a list of aftermarket devices
+     * Caller must be the owner of the aftermarket device's parent manufacturer or an authorized address
+     * The manufacturer node owner must grant the MANUFACTURER_FACTORY_RESET_PRIVILEGE privilege to the authorized address
+     * @param adIdAddrs List of deviceId-deviceAddress pairs to be set
+     */
+    function setAftermarketDeviceAddressByManufacturerBatch(
```

Comment:
Maybe we should call it `resetAftermarketDeviceAddressByManufacturerBatch`? Just to match the event?

Reply:
Or we could rename de event.